pkg/commands/run.go

```diff
@@ -193,6 +193,10 @@ func initFlagSet(fs *pflag.FlagSet, cfg *config.Config, m *lintersdb.Manager, is
 		"Depguard: check list against standard lib")
 	hideFlag("depguard.include-go-root")
 
+	fs.StringSliceVar(&lsc.Depguard.IgnoreFileRules, "depguard.ignore-file-rules", nil,
+		"Depguard: rules to ignore certain files for consideration")
+	hideFlag("depguard.ignore-file-rules")
+
```

Comment:
Can you remove this? We don't add new flags.

https://github.com/golangci/golangci-lint/blob/29eedbf6e56508cf1ca52ad0eeed067b1c3a3066/pkg/commands/run.go#L128